### ISSUE TITLE
attribute error looking for pkg_resources.SetuptoolsVersion

### ISSUE THREAD

**Issue by arterrey** [buildout/buildout#442]
`pkg_resources.SetuptoolsVersion` is now gone (see https://github.com/pypa/setuptools/commit/eeeb9b27fa48fccf2b5d52919eff1c75c4ad1718) and has been renamed  to `SetuptoolsLegacyVersion`

Using setuptools 39.0.1 and zc.buildout 2.11.1
buildout now breaks with an attribute error...

```
bin/buildout
Traceback (most recent call last):
  File "bin/buildout", line 11, in <module>
    load_entry_point('zc.buildout==2.11.1', 'console_scripts', 'buildout')()
  File "/a/lib/python3.6/site-packages/pkg_resources/__init__.py", line 480, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/a/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2693, in load_entry_point
    return ep.load()
  File "/a/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in load
    return self.resolve()
  File "/a/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2330, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/a4/lib/python3.6/site-packages/zc.buildout-2.11.1-py3.6.egg/zc/buildout/buildout.py", line 18, in <module>
    import zc.buildout.easy_install
  File "/a/lib/python3.6/site-packages/zc.buildout-2.11.1-py3.6.egg/zc/buildout/easy_install.py", line 45, in <module>
    pkg_resources.SetuptoolsVersion(setuptools_version) >=
AttributeError: module 'pkg_resources' has no attribute 'SetuptoolsVersion'
```

**Comment by enkidulan** [comment]
same here, pinnig buildout version to `2.9.6` fix the problem. Issue appers starting with version `2.10.0`

**Comment by gforcada** [comment]
@enkidulan I think that this is rather a problem with newer setuptools. On version 39.0.0 ``SetuptoolsVersion`` got removed https://setuptools.readthedocs.io/en/latest/history.html#v39-0-0

**Comment by enkidulan** [comment]
@gforcada in my case buildout version `2.9.6` works with the newest `setuptools` version fine. It seems like a combination :) Yep, here are new changes introduced in `2.10.0` that depend on `SetuptoolsVersion` -
https://github.com/buildout/buildout/compare/2.9.6...2.10.0?expand=1#diff-784f9bb654bbcee96dcbc547bbddd493R45

**Comment by gforcada** [comment]
@enkidulan yes, I saw those changes, but I'm rather busy right now to try to cook a patch (I've never done that before for zc.buildout).

**Comment by enkidulan** [comment]
I've just created a PR that should fix the issue - https://github.com/buildout/buildout/pull/443

**Comment by reinout** [comment]
Thanks for the fix, I've released 2.11.2!

Does that fix everyone's issue?

**Comment by arterrey** [comment]
Cheers @reinout :smiley: really appreciate that. Tested with 2.11.2 and it is now working again.
